src/views/ProjectView.vue

```diff
@@ -17,6 +17,7 @@ const isPopupVisible = ref(false)
 const uniqueDataSessions = ref([])
 const newSessionName = ref('')
 const errorMessage = ref('')
+const apiBaseUrl = 'http://127.0.0.1:8000/api/datasessions/'
```

Comment:
Don't need to do this now, but we might want to create a story to put these endpoints in a centralized place, like PTR does - maybe from environment variables? :D 

https://github.com/LCOGT/ptr_ui/blob/dev/src/store/modules/api_endpoints.js

Reply:
100% yes